src/Scene/DefaultUpdater.js

```diff
@@ -0,0 +1,32 @@
+/**
```

Comment:
The idea is to have a generic `updater` or 'browseTree' to which have added some static functions `nodeProcess`, but not having several updater. The 'nodeProcess' are applied to each node and the traverse stops when a process returns false or undefined. There is also a notion of 'nodeProcess' order.

**example**
```js
const process = [
  nodeProcess.occlusion,
  nodeProcess.occlusionHorizon,
  nodeProcess.SSE,
  nodeProcess.refineNode,
];
var updaterQuadtree = new updater(process);
var quadTree = new QuadTree();

// link updater to quadtree for update 
scene.apply(quadTree,updaterQuadtree);
```
We can even imagine a process tree to be applied to each node. The children process will be performed according to the parent process.